deploy-manage/monitor/autoops/cc-connect-self-managed-to-autoops.md

```diff
@@ -70,7 +70,8 @@ If you don’t have an existing {{ecloud}} account:
 :sync: existing
 
 If you already have an {{ecloud}} account:
-1. Log in to [{{ecloud}}](https://cloud.elastic.co?page=docs&placement=docs-body).
+1. Log in to [{{ecloud}}](https://cloud.elastic.co/login?redirectTo=%2Fconnect-cluster-services).
+    - The link provided should take you directly to the **Connect your self-managed cluster** page
 2. On your home page, in the **Connected clusters** section, select **Connect self-managed cluster**. 
```

Comment:
As part of this update, should we remove step 2 if we say that the link in step 1 brings the user to the  **Connect your self-managed cluster** page?

Reply:
we should probably keep both, as some people will just want to follow instructions instead of using our link